src/config/eslint.config.json

```diff
@@ -1,4 +1,5 @@
 {
+	"parser": "babel-eslint",
```

Comment:
https://stackoverflow.com/questions/36001552/eslint-parsing-error-unexpected-token is the issue that this addresses